src/client.rs

```diff
@@ -124,6 +124,38 @@ fn validate_sequence_set(
     Ok(value)
 }
 
+/// Lovingly borrowed from the cargo crate
+///
+/// Joins an iterator of [std::fmt::Display]'ables into an output writable
+pub fn iter_join_onto<W, I, T>(mut w: W, iter: I, delim: &str) -> std::fmt::Result
```

Comment:
These should definitely not be `pub` :sweat_smile: Can you change them to `pub(crate)`?

Reply:
oh.. good catch.